src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -444,7 +444,7 @@ display: "block"
 
 Determines when a font must be downloaded and used based on a specific [range of unicode characters](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range). If a character on the page matches the configured range, the browser will download the font and all characters will be available for use on the page. To configure a subset of characters preloaded for a single font, see the [subsets](#subsets) property instead.
 
-This can be useful for localization to avoid unnecessary font downloads when a specific part of your website uses a different alphabet and will be displayed with a separate font. For example, a website that offers both English and Japanese versions can prevent the browser from downloading the Japanese font on English versions of the page that do not contain any of the Japanese characters provided in `unicodeRange`.
+This can be useful for localization to avoid unnecessary font downloads when a specific part of your website uses a different language and will be displayed with a separate font. For example, a website that offers both English and Japanese versions can prevent the browser from downloading the Japanese font on English versions of the page that do not contain any of the Japanese characters provided in `unicodeRange`.
```

Comment:
I think the word "alphabet" was chosen specifically to refer to characters to be displayed, since we're talking about fonts. (Since e.g. English and French use mostly the same *alphabet*, just with some additional characters for accented letters in French.)

So I think alphabet may be a more precise distinction here, but I'll let Florian decide!

Reply:
I'm not an expert of this CSS property but I believe alphabet is right there